package/yast2-storage-ng.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Wed Aug 30 15:44:35 UTC 2017 - jlopez@suse.com
+
+- Added overall summary for the expert partitioner.
+- Refac code for tables.
```

Comment:
"Refac" is not a word. Other than that, it was done as part of the job for the previous sentence and I don't see any value for the user per se, so I would simply remove this line.